schemas/pci/pci-bus.yaml

```diff
@@ -122,6 +122,12 @@ properties:
     description: GPIO controlled connection to PERST# signal
     maxItems: 1
 
+  slot-power-limit-milliwatt:
+    description:
+      If present, specifies slot power limit in milliwatts.
+      This property is invalid in host bridge nodes.
+    $ref: "types.yaml#/definitions/uint32
```

Comment:
'-milliwatt' needs to be added to schemas/property-units.yaml and then the type $ref can be dropped. However, we already have '-microwatt' in use (op-sink-microwatt) though also not documented in property-units.yaml, so can we use microwatt instead? The reason for not supporting any unit prefix is so that we don't end up with random scales for the same things.

Reply:
[We already have milliwatt for sfp nodes](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/Documentation/devicetree/bindings/net/sff,sfp.txt?h=v5.16-rc8), and this is already used by many dts, so it is not going away. Should both milli and micro be added to property-units ?

Reply:
Ah yes, I missed that despite grepping for just 'watt'. We should add both to property-units.yaml

Reply:
Ok, so in that case I will use milliwatt for this pci-bus.